e2e/cypress/integration/cloud/onboarding/cloud_sysadmin_onboarding_spec.js

```diff
@@ -0,0 +1,420 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import {testWithConfig} from '../../../support/hooks';
+
+const adminSteps = ['complete_profile', 'team_setup', 'invite_members', 'hide'];
+
+describe('Cloud Onboarding - Sysadmin', () => {
+    let townSquarePage;
+    let sysadmin;
+
+    testWithConfig({
+        ServiceSettings: {
+            ExperimentalChannelSidebarOrganization: 'default_on',
```

Comment:
Do you specifically need the new sidebar for this? If so, we might want to talk about getting that pushed to default_on for others since we haven't really discussed that on our side recently.

Reply:
Yeah, the sidebar module is only for the new sidebar (which I assume is what we're going to launch with Cloud). But yeah that's what I'm hoping for.

I think it can be turned on for the other sidebar if need be though, I'll bring it up in the Cloud meeting today to determine the plan.